test/file04.ts

```diff
@@ -0,0 +1,22 @@
+import styled from '@emotion/styled';
+import { Theme } from '@xsite-ui/common.style';
+
+export const Container = styled.div<{ theme?: Theme }>`
+  display: grid;
+  gap: ${({ theme }) => theme?.space?.[5] || 20}px;
+  grid-template-columns: repeat(2, 1fr);
+
+  & [data-role='question'] {
+    grid-column: 1 / span 2;
+  }
+
+  & [data-role='question']:nth-of-type(4) {
```

Comment:
Using CSS pseudo-class selector (nth-of-type). May not be well supported in older browsers. Consider using a class or data attribute instead.